src/SimpleList/manifest.json

```diff
@@ -340,6 +321,125 @@
           "default": true
         }
       ]
+    },
+    {
```

Comment:
Wherever the list header child component currently is, let's leave it there. The mockup may have been wrong on this one.